axiom/nr_limits.h

```diff
@@ -33,9 +33,14 @@
 #define NR_MAX_SEGMENTS 2000
 
 /*
- * The maximum number of span events in a transaction.
+ * The default maximum number of span events in a transaction.
  */
-#define NR_MAX_SPAN_EVENTS 1000
+#define NR_SPAN_EVENTS_DEFAULT_MAX_SAMPLES_STORED 2000
+
+/*
+ * The absolute maximum number of span events in a transaction.
+ */
+#define NR_SPAN_EVENTS_MAX_SAMPLES_STORED 10000
```

Comment:
Wouldn't following macro names better reflect their purpose:

```
#define NR_SPAN_EVENTS_MAX_SAMPLES_STORED_DEFAULT 2000
#define NR_SPAN_EVENTS_MAX_SAMPLES_STORED_MAXIMUM 10000
```

Reply:
Good point, it would make more sense at the end or beginning.
Looking at the file, the convention seems to be to move it to the beginning (after the NR_).
Maybe it should be something like the following to follow the convention:
```
#define NR_DEFAULT_SPAN_EVENTS_MAX_SAMPLES_STORED 2000 
#define NR_MAX_SPAN_EVENTS_MAX_SAMPLES_STORED 10000
```

Reply:
Agreed I am leaning towards the option following the existing convention with it at the beginning.

Reply:
Refactored with [5e663505](https://github.com/newrelic/newrelic-php-agent/pull/384/commits/5e66350575628bf34f3045a2b39f5ccd471a21d6).